repl/src/main/scala/ammonite/repl/interp/Interpreter.scala

```diff
@@ -56,6 +56,8 @@ class Interpreter(prompt0: Ref[String],
       |""".stripMargin
 
   val SheBang = "#!"
+  var lastEx:Throwable=null
```

Comment:
Internal variables should be named the same way as external variables =) We shouldn't cut corners on readability just because it's internal